derpycode/cgenie.muffin
# addition of flexible C:P + manuscript experiments

The actual code changes are relatively restricted, and enable  flexible C:P stiochiometry to be selected.
Everything else is the addition of new (glacial) boundary conditions, user- or base-config files, and forcing files -- all described in the genie-userconfigs/MS/odalenetal.BG.2019/readme.txt file.

To test:

(1) visually check the code changes

(2) checkout the branch and make testbiogem

(3) repeat the first 1000 years of the 3 key spin-ups as described in the readme.txt file:

./runmuffin.sh malod.worjh2.pref.BASEFe /MS/odalenetal.BG.2019 malod.worjh2.PO4Fe.pref.(VERSION).SPIN 10000
where (VERSION) should be replaced by one of the model version abbreviations RED, GAM or 121.

Given below is the Total POC export value (biogem_year_0999_500_diag_GLOBAL_AVERAGE.res) for the year 999.5 centred annual average:

121 --  Total POC export   :   0.5194650E+15 mol yr-1 =   6.234 GtC yr-1
GAM --  Total POC export   :   0.5637508E+15 mol yr-1 =   6.765 GtC yr-1
RED --  Total POC export   :   0.4653579E+15 mol yr-1 =   5.585 GtC yr-1

These 1000 year experiment results were run on a modified svn checkout.
There is no reason why the results should not be identical to the stated number of significant figures.
